contrib/pinot-druid-migration/src/main/java/org/apache/pinot/druid/tools/DruidToPinotSegmentConverter.java

```diff
@@ -31,9 +31,9 @@
 public class DruidToPinotSegmentConverter {
   private static String _pinotSchemaFilePath;
```

Comment:
We usually use either `private` (instance variable) or `private static final` (constant). Let's actually remove these variables.

We can do either of the followings:
1. Do everything in `main()`
2. Add variables to `convertSegment()` function